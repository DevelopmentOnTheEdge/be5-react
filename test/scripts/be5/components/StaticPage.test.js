import React from 'react';
import renderer from 'react-test-renderer';
import StaticPage from '../../../../src/scripts/be5/components/staticPage';

test('snapshot', () => {
    const pageContent = "<h1>Static page<h1>test content"
    const tree = renderer.create(
        <StaticPage value={pageContent} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
