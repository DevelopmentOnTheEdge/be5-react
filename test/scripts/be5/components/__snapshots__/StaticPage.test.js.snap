// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="staticPage"
  dangerouslySetInnerHTML={
    Object {
      "__html": "<h1>Static page<h1>test content",
    }
  }
/>
`;
